draft-ietf-quic-transport.md

```diff
@@ -2721,6 +2721,66 @@ between endpoints.  Application protocols SHOULD define rules for handling
 streams that are prematurely cancelled by either endpoint.
 
 
+# Alternative Initial Set {#alternative-initial}
+
+In order to avoid ossification of the cleartext and obfuscated fields of QUIC
+packets, a server can announce an alternative set of initial values to be used,
+which is comprised of:
+
+* Alternative version number; a 32-bit unsigned number that is to be presented
+  on wire in place of the version number specified in this document.  This value
+  MUST NOT be a reserved version ({{versions}}).
+
+* Packet type modifier; a two-bit value that is to be applied as a bit-wise
+  exclusive or (XOR) to the most significant bits of the Initial, Handshake,
```

Comment:
Shouldn't this be `to the Long Packet Type field of` instead of `to the most significant bits of` ?

Reply:
Definitely Yes! I missed the fact that we had a name for the two bits. Thank you.

Reply:
We have two choices for the placement of these bits in the NEW_TOKEN.  0bXXTTXXXX or 0bXXXXXXTT.  The former might be easier to apply, the latter might be easier to explain.

Reply:
Fixed in 7eded0a.